packages/web/src/scss/components/Accordion/_Accordion.scss

```diff
@@ -0,0 +1,97 @@
+@use 'sass:map';
+@use 'theme';
+@use '../Collapse';
```

Comment:
Remove, see comment below.

Reply:
Solved by replacing the class for `.Collapse` and `.Collapse__content`